cmd/create.go

```diff
@@ -20,18 +20,25 @@ var (
 var componentCreateCmd = &cobra.Command{
 	Use:   "create <component_type> [component_name] [flags]",
 	Short: "Create new component",
-	Long: `Create new component.
-If component name is not provided, component type value will be used for name.
-	`,
-	Example: `  # Create new nodejs component with the source in current directory. 
+	Long: `Create new component to deploy on OpenShift.
+
+If component name is not provided, component type value will be used for the name.
+
+A full list of component types that can be deployed is available using: 'ocdev component list'`,
+	Example: `  # Create new Node.js component with the source in current directory. 
   ocdev create nodejs
 
-  # Create new nodejs component named 'frontend' with the source in './frontend' directory
+  # Create new Node.js component named 'frontend' with the source in './frontend' directory
   ocdev create nodejs frontend --local ./frontend
 
-  # Create new nodejs component with source from remote git repository.
+  # Create new Node.js component with source from remote git repository.
   ocdev create nodejs --git https://github.com/openshift/nodejs-ex.git
-	`,
+
+  # Create a Ruby component
+  ocdev create ruby
+	
+	# Create a Python component
```

Comment:
indentation. There should be two spaces at the beginning to alight it with everything else.

Reply:
stupid vim haha

Reply:
woops vim